packages/popover/src/PopoverContent.js

```diff
@@ -131,12 +140,11 @@ const PopperGuide = styled(Box)`
   box-sizing: border-box;
 `
 const ContentContainer = styled.section`
-  box-shadow: 0 0 0 1px ${({ theme, borderColor }) => theme.colors[borderColor]},
+  box-shadow: 0 0 0 1px ${props => getPaletteColor(props.borderColor)},
```

Comment:
I think this needs to be `${props => getPaletteColor(props.borderColor)(props)}`

Reply:
Fixed!